services/worker/vendors/trec-car-tools/python3/conf.py

```diff
@@ -27,7 +27,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.coverage',
+    # 'sphinx.ext.coverage',
```

Comment:
I never changed anything in `services/worker/vendors/trec-car-tools`, it's an exact copy of the dependency... Not sure if we should change it.

Anyway, maybe we should delete this dependency anyway

Reply:
Also: it seems to break the worker tests

Reply:
I was trying to have our CI green, but it didn't work anyway.